sdk/communication/communication-administration/review/communication-administration.api.md

```diff
@@ -133,6 +133,7 @@ export type GetAreaCodesOptions = OperationOptions;
 // @public
 export interface GetAreaCodesRequest {
     countryCode: string;
+    locationOptionsQueries: LocationOptionsQueries;
```

Comment:
```typescript
// @public
export interface LocationOptionsQuery {
    labelId?: string;
    optionsValue?: string;
}
```
^ those two props shouldn't be optional right? Another bug.

Reply:
yep another one